autospec/specfiles.py

```diff
@@ -1555,12 +1555,9 @@ def write_ant_pattern(self):
         self.write_build_prepend()
         self.write_proxy_exports()
         self._write_strip("export ANT_HOME=/usr/share/ant")
-        self._write_strip("ant dist")
+        self._write_strip("ant -d -v " + self.extra_make)
         self._write_strip("%install")
         self.write_install_prepend()
-        jar_dir = os.path.join("%{buildroot}/usr/share/jar", self.name)
-        self._write_strip(f"mkdir -p {jar_dir}")
-        self._write_strip(f"install -m 0644 dist/*.jar {jar_dir}")
```

Comment:
Are we going to use `install_append` exclusively for packages that build with Ant? Or is the install process TBD?

Reply:
Exclusively, yes. Artifacts produced vary too much, and for some packages, we'll want to install the JAR, while for others, we'll want to install regular files and _not_ the JAR.

Reply:
/me reads the PR summary... sorry for reading that very carefully.

Sounds good to me.